contents/blog/10x-engineers-do-user-interviews.md

```diff
@@ -0,0 +1,138 @@
+---
+date: 2023-02-19
+title: Becoming a 10x engineer - why user interviews are key
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - luke-harries
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product engineers
+---
+
+Most people think becoming a 10x engineer means improving their typing speed, buying more monitors, or using the latest framework where everything is a macro. These hacks might speed up the time from an idea to a rolled-out implementation, but you'll get a 1.5x boost at most.
+
+So, where does the next 8.5x come from? By zooming out and solving for user impact. You, as an engineer, need to deeply understand (and care) about your users. In doing so, you'll generate true leverage by choosing the right problem, building the right solution, and solving their "hair-on-fire" problems. The best way to do this? Speak directly to them.
+
+## Tickets are highly lossy
+
+Your users are real people. They have problems to overcome. They work in a team. They have hopes and dreams. But if the only information that you get is the ticket "Add a button to this page" you miss all that context. And you miss the opportunity to build a significantly better product.
+
+Let's discuss an example from PostHog: we were getting requests for setting currency symbols on charts. We could have added the most common currency symbols to the chart and called it done.
+
+But we decided to dig deeper and speak to our users. Why did they want currency symbols on their charts in PostHog? What was the problem?
+
+We learned that if a teammate created the chart it was often unclear what the axes of charts referred. This affected charts containing revenue but also other types of data such as app response times measured in milliseconds. We reframed the problem to support custom prefixes and postfixes and built it within a day. And compared to the original problem definition we solved the long-tail of use cases that we'd otherwise have to add incrementally.
```

Comment:
I don't feel this example is particularly strong - what do you think @andyvan-ph? Quite a minor product. I can try and find a better on

Another example we could do is the origin of session recording as a way to better answer what exact users are doing?

Reply:
I've added a much better example